specs/Slider.md

```diff
@@ -113,13 +112,13 @@ https://developer.microsoft.com/en-us/fabric#/controls/web/slider
 ### Differences of Fabric/Stardust to resolve
 
 | Name                        | Fx  | Recommendation                                                                                                                                                                              |
-| --------------------------- | --- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| --------------------------- | --- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
 | accessibility               | S   | Discuss: In its current state it's unclear why a user would ever pass an accessibility behavior in. This is different from every framework. How does this help developers?                  |
 | animation                   | S   | Remove: Why would a user need this as a prop on Slider? Also ambiguous; animation for slider movement or just css on the root?                                                              |
 | ariaLabel                   | F   | Discuss: Without having the user read aria specs, can we abstract this into `screenReaderDescription` or `accessibility.description`? At minimum we need an example of what should go here. |
 | ariaValueText               | F   | `ariaLabel` set label for the slider, the ariaValueText set string instead of slider number value                                                                                           |
 | buttonProps                 | F   | Replace: Use slotProps/slots.                                                                                                                                                               |
-| componentRef                | F   | Discuss: Slider imperative API to set focus and read value. Do we need these?                                                                                                               |  |
+| componentRef                | F   | Discuss: Slider imperative API to set focus and read value. Do we need these?                                                                                                               |     |
```

Comment:
Change in line 115 is happening because of this line, it should be:

```suggestion
| componentRef                | F   | Discuss: Slider imperative API to set focus and read value. Do we need these?                                                                                                               |
```